heron/simulator/src/java/com/twitter/heron/simulator/instance/SpoutInstance.java

```diff
@@ -78,7 +79,7 @@ protected void produceTuple() {
       }
 
       if (collector.getTotalDataEmittedInBytes() - totalDataEmittedInBytesBeforeCycle
```

Comment:
Just curious since I don't have a strong opinion here: in this kind of scenario, shall we use [greaterThan](https://github.com/twitter/heron/blob/master/heron/common/src/java/com/twitter/heron/common/basics/ByteAmount.java#L179) API instead?